site/tests/integration/blockpage.test.ts

```diff
@@ -0,0 +1,113 @@
+import { expect, test } from "playwright-test-coverage";
+
+import blocksData from "../../blocks-data.json";
+
+const codeBlock = blocksData.find(
+  ({ name }) => name === "@hashintel/block-code",
+);
+
+const unsupportedBlock = blocksData.find(
+  ({ name }) => name === "@hashintel/block-embed",
+);
+
+if (!codeBlock || !unsupportedBlock) {
+  throw new Error("Code and Embed blocks need to be prepared before tests run");
+}
+
+test("Block page should contain key elements", async ({ page, isMobile }) => {
+  await page.goto(codeBlock.blockPackagePath);
+
+  await expect(
+    page.locator(`h1:has-text('${codeBlock.displayName}')`),
+  ).toBeVisible();
+
+  await expect(
+    page.locator(`p:has-text('${codeBlock.description}')`),
+  ).toBeVisible();
+
+  await expect(page.locator(`text=@${codeBlock.author}`)).toBeVisible();
+  await expect(page.locator(`text=@${codeBlock.author}`)).toHaveAttribute(
+    "href",
+    `/@${codeBlock.author}`,
+  );
+  await expect(page.locator(`text=V${codeBlock.version}`)).toBeVisible();
+
+  const blockExample = codeBlock.examples[0] as Record<string, string>;
+
+  if (isMobile) {
+    await page.locator("text=Source Code").click();
+    await expect(page.locator("text=Data Source")).toBeVisible();
+  } else {
+    await expect(page.locator("text=Block Properties")).toBeVisible();
+  }
+  await expect(page.locator("#simple-tabpanel-0 >> textarea")).toHaveValue(
+    JSON.stringify(blockExample, null, 2),
+  );
+
+  if (isMobile) {
+    await page.locator("text=Data Source").click();
+    await page.locator("text=Block Schema").click();
+  } else {
+    await page.locator("text=Block Schema").click();
+  }
+  await expect(page.locator("#simple-tabpanel-1")).toBeVisible();
+  // await expect(page.locator("#simple-tabpanel-1")).not.toBeEmpty();
```

Comment:
👀

Reply:
Uncommented in [5adad49](https://github.com/blockprotocol/blockprotocol/pull/450/commits/5adad49fdf2865ff470f34040be638718ea3a2aa)